.travis.yml

```diff
@@ -155,7 +155,15 @@ jobs:
         name: ws3
         paths:
           - coverage.*.xml
-      # Inherits "global" job phases defined below (e.g. before_install, install, before_script, script, etc.)
+    script:
```

Comment:
I think it would be better to figure out why there are tests that are now going longer than 10 minutes without output than to change this timeout setting.

If we do determine there are scenarios that will results in these logging gaps, we may want to update the default `script` instead of these overrides.

Reply:
The main one that was going longer than 10 minutes without output was the load_to_from_delta one. I added the travis_wait in the other one as a precautionary step but it doesn't seem to have any affects now.

Reply:
Looking at other builds using 3.8.16 I see that the load_to_from_delta and other select jobs almost always take closer to 20 minutes so I think keeping this setting doesn't do us any harm